src/routes/dictionaries/+page.svelte

```diff
@@ -56,28 +55,17 @@
         data.user,
         sequenceAdaptationName,
       );
-
-      if (uploadedDictionaryOrAdaptation === null) {
-        throw Error('Failed to upload file');
+      if ('adaptation' in uploadDictionaryOrAdaptation) {
```

Comment:
Now that I see this again, can we move the toasts into the effect function? We don't normally show toasts from within components themselves

Reply:
I moved the toast into the effects.ts